src/component/mxgraph/shape/render/IconPainter.ts

```diff
@@ -280,52 +280,25 @@ export default class IconPainter {
   /**
    * This icon is used by `compensation event`.
    */
-  public paintDoubleLeftArrowheadsIcon({ c, ratioFromParent, setIconOrigin, shape, icon }: PaintParameter): void {
-    // implementation adapted from https://thenounproject.com/vityavorobyev/icon/443401/
-    const canvas = this.newBpmnCanvas({ c, ratioFromParent, setIconOrigin, shape, icon }, { height: 55, width: 100 });
+  public paintCompensationIcon({ c, ratioFromParent, setIconOrigin, shape, icon }: PaintParameter): void {
```

Comment:
I approve too quickly. Can you let the method name as it was ?
```suggestion
  public paintDoubleLeftArrowheadsIcon({ c, ratioFromParent, setIconOrigin, shape, icon }: PaintParameter): void {
```